src/database/dynamic-update-query.ts

```diff
@@ -0,0 +1,18 @@
+export function buildUpdateQuery(table: string, data: any, where: Map<string, string>, returnUpdatedRow: boolean = false) {
+    const keys = Object.keys(data);
+    const values = Object.values(data);
+    const setString = keys.map((key, i) => `${key} = $${i + 1}`).join(', ');
+
+    const whereKeys = Array.from(where.keys());
+    const whereValues = Array.from(where.values());
+    const whereString = whereKeys.map((key, i) => `${key} = $${values.length + i + 1}`).join(' AND ');
+
+    values.push(...whereValues);
+
+    const query = {
```

Comment:
there is a possibility of mal-formed query if where map is empty or null, you should add checks to handle those cases

Reply:
Taken care